EmpleoDotNet.Core/Domain/JobOpportunityLocation.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EmpleoDotNet.Core.Domain
+{
+    public class JobOpportunityLocation
+    {
+        public int Id { get; set; }
+        public string PlaceId { get; set; }
+        public string Name { get; set; }
+        public string Latitude { get; set; }
+        public string Longitude { get; set; }
+    }
+}
```

Comment:
Change this name implies in rebuild the migration and Stored procedure to update data in production, so, we can use this name for the moment and then update it when is necessary.